charts/argo-cd/templates/argocd-application-controller/clusterrolebinding.yaml

```diff
@@ -1,5 +1,4 @@
-{{- $config := .Values.controller.clusterAdminAccess | default dict -}}
-{{- if hasKey $config "enabled" | ternary $config.enabled .Values.createClusterRoles }}
```

Comment:
Please restore the change on the application-controller. We still need backward compat on this one.